src/test/suites/e2e/test8.js

```diff
@@ -0,0 +1,26 @@
+import HomePage from '../../../app/pages/HomePage';
```

Comment:
I can see that you have used the test case number with test keyword, but can we have an appropriate test case name

Reply:
done